integration-tests/foundation-grouped/pom.xml

```diff
@@ -112,12 +112,10 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-log</artifactId>
         </dependency>
-        <!-- TODO: https://github.com/apache/camel-quarkus/issues/5995
```

Comment:
Please can you restore the commented out `camel-quarkus-microprofile-fault-tolerance` dependency. #5995 isn't properly solved until we get Camel 4.6.0.

I'll try to find a way to stop the auto regeneration of the pom.xm from adding it back.

Reply:
Done.